init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+: ${KUBERNETES_NODE_NAME:?"You need to set the KUBERNETES_NODE_NAME environment variable."}
+: ${CALICO_ROUTE_REFLECTOR_CLUSTER_ID:?"You need to set the CALICO_ROUTE_REFLECTOR_CLUSTER_ID environment variable."}
+: ${CALICO_ROUTE_REFLECTOR_LABEL:?"You need to set the CALICO_ROUTE_REFLECTOR_LABEL environment variable."}
+
+echo "Labelling node \"${KUBERNETES_NODE_NAME}\" with route reflector label..."
+kubectl label node --overwrite "${KUBERNETES_NODE_NAME}" "${CALICO_ROUTE_REFLECTOR_LABEL}="
```

Comment:
Is the `=` at the end required? If yes, for what?

Reply:
Yes, it is required as the `kubectl label node` command requires labels in the format of `key=[value]` where `value` can be an empty string.

For example, the following:

```console
kubectl label node ip-10-24-101-78.ec2.internal calico-route-reflector=
```

Labels node `ip-10-24-101-78.ec2.internal` as follows:

```console
$ kubectl describe node ip-10-24-101-78.ec2.internal
Name:               ip-10-24-101-78.ec2.internal
...
Labels:            calico-route-reflector=
...
```

Reply:
Noted.